tools/sggolangcilint/golangci.yml

```diff
@@ -122,6 +118,10 @@ linters:
     # [fast: true, auto-fix: false]
     - promlinter
 
+    # Reports direct reads from proto message fields when getters should be used.
+    # [fast: true, auto-fix: true]
```

Comment:
I actually don't know what `fast: true` means, or where I can get the information whether `protogetter` is fast or not.

Reply:
@fredrikaverpil 

```bash
 $ golangci-lint linters | grep protogetter
protogetter: Reports direct reads from proto message fields when getters should be used [fast: false, auto-fix: true]
```